src/Legend/Legend.tsx

```diff
@@ -134,11 +155,19 @@ export class Legend extends React.Component<LegendProps, LegendState> {
         className={finalClassName}
         {...passThroughProps}
       >
-        <img
-          src={this.state.legendUrl}
-          alt={alt}
-          onError={this.onError.bind(this)}
-        />
+        {this.state.legendError ?
+          <div
+            className='legend-load-error'
```

Comment:
where are the rules for this class defined?

if no rules are specified, please remove the class.

Reply:
its a bonus for external applications so they can modify the error div to their likings